.prow.yaml

```diff
@@ -220,6 +220,56 @@ presubmits:
             items:
               - key: nephio.yaml
                 path: nephio.yaml
+  - name: e2e-oai-ubuntu-jammy
```

Comment:
What about fedora34 support?

Reply:
I was thinking to complete the OAI e2e testing with Ubuntu Jammy first and eventually try with Fedora, more likely the latest stable release

Reply:
@electrocucaracha Sounds good.
cc: @dkosteck